src/nvim/mark.c

```diff
@@ -1522,3 +1522,9 @@ void copy_viminfo_marks(vir_T *virp, FILE *fp_out, int count, int eof, int flags
   }
   xfree(name_buf);
 }
+
+// Export mark.h's inline functions here to make them usable when not inlined.
+extern inline bool lt(pos_T a, pos_T b);
+extern inline bool equalpos(pos_T a, pos_T b);
+extern inline bool ltoreq(pos_T a, pos_T b);
+extern inline void clearpos(pos_T *a);
```

Comment:
Instead of doing this, perhaps you could do what was done in #2486 and use `REAL_FATTR_ALWAYS_INLINE`.